src/lib/store.js

```diff
@@ -269,12 +269,15 @@ export default class Store {
 		}
 	};
 
-	selectAllVendorLegitimateInterests = (isSelected, update = true) => {
-		const {vendorLegitimateInterests} = this.tcModel;
-		const vendorsWithLegIntsIds = Object.keys(this.vendorList.vendors)
+	getVendorsWithLegints = () => Object.keys(this.vendorList.vendors)
```

Comment:
Does this function return vendors or just theirs ids? If it returns only ids, I think name of the function should be change to something like getVendorsWithLegIntsIds.